guide/src/reference/deployment.md

```diff
@@ -14,6 +14,7 @@ should behave the same way in this respect. The values possible here are:
 | [`bundler`]     | Suitable for loading in bundlers like Webpack              |
 | [`web`]         | Directly loadable in a web browser                         |
 | [`nodejs`]      | Loadable via `require` as a Node.js module                 |
+| [`Deno`]        | Loadable using imports from Deno modules                   |
```

Comment:
I think this wants to be a lower-case `d`, and it looks like the destination of this link needs to be filled in?

Reply:
it looks like the branding is an uppercase D: https://deno.land/
I will fix the link.

Reply:
though this is not the branding but the flag, so nevermind